.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,62 @@
+name: 🐛Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: 🧐 What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    validations:
+      required: true
+    attributes:
+      label: "👟 Reproduction steps"
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: "When I ..."
+  - type: dropdown
+    id: version
```

Comment:
please also add Utopia Preloader Version option.

Reply:
what will be the option in for Utopia Preloader Version dropdown ?